File: be/dtos/request.go
package dtos

type RequestLoginUser struct {
	Email    string `json:"email" binding:"required,email"`
	Password string `json:"password" binding:"required,min=5"`
}

type PenerimaanBarang struct {
	WhsIdf           int    `json:"whs_idf" binding:"required"`
	TrxInDate        string `json:"trx_in_date" binding:"required"`
	TrxInSuppIdf     int    `json:"trx_in_supp_idf" binding:"required"`
	TrxInNotes       string `json:"trx_in_notes" binding:"required"`
	TrxInDProductIdf string `json:"trx_in_dproduct_idf" binding:"required"`
	TrxInDQtyDus     string `json:"trx_in_dqty_dus" binding:"required"`
	TrxInDQtyPcs     string `json:"trx_in_dqty_pcs" binding:"required"`
}

File: fe/src/@types/request.ts
export interface PenerimaanBarang {
    whs_idf: number;
    trx_in_date: string;
    trx_in_supp_idf: number;
    trx_in_notes: string;
    trx_in_dproduct_idf: number;
    trx_in_dqty_dus: number;
    trx_in_dqty_pcs: number;
}

export interface PengeluaranBarang {
    whs_idf: number;
    trx_out_date: string;
    trx_out_supp_idf: number;
    trx_out_notes: string;
    trx_out_dproduct_idf: number;
    trx_out_dqty_dus: number;
    trx_out_dqty_pcs: number;
}
